tasks/configure-fpm.yml

```diff
@@ -69,9 +69,16 @@
   when: php_enable_php_fpm
   notify: restart php-fpm
 
+- name: systemd restart php-fpm
+  systemd:
+    name: "{{ php_fpm_daemon }}.service"
+    daemon_reload: yes
+    state: restarted
```

Comment:
Always restarting this service would cause the role's idempotence to break. It would need to only be done if triggered via a `notify` or something like that.

Also, the `service` module should automatically work with both initv and systemd, depending on what's available—so I still wonder if it would be necessary to do this. Maybe only if the task that copies the unit file reports `changed`?